[06/17/2017 14:13:02] Authentication to host 'localhost' for user 'root' using method 'mysql_native_password' failed with message: Unknown database '\\192.164.0.127\sample\sample'
[06/17/2017 14:13:02] Database: C@TCHM3
[06/17/2017 14:25:38] Unable to connect to any of the specified MySQL hosts.
[06/17/2017 14:25:38] Database: C@TCHM3
[06/17/2017 14:53:11] Authentication to host 'localhost' for user 'blade' using method 'mysql_native_password' failed with message: Access denied for user 'blade'@'localhost' (using password: YES)
[06/17/2017 14:53:11] Database: C@TCHM3
[06/17/2017 14:53:12] LoadSQL - MySql.Data.MySqlClient.MySqlException (0x80004005): Authentication to host 'localhost' for user 'blade' using method 'mysql_native_password' failed with message: Access denied for user 'blade'@'localhost' (using password: YES) ---> MySql.Data.MySqlClient.MySqlException (0x80004005): Access denied for user 'blade'@'localhost' (using password: YES)
   at MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   at MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.AuthenticationFailed(Exception ex)
   at MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.ReadPacket()
   at MySql.Data.MySqlClient.Authentication.MySqlAuthenticationPlugin.Authenticate(Boolean reset)
   at MySql.Data.MySqlClient.NativeDriver.Authenticate(String authMethod, Boolean reset)
   at MySql.Data.MySqlClient.NativeDriver.Open()
   at MySql.Data.MySqlClient.Driver.Open()
   at MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   at MySql.Data.MySqlClient.MySqlPool.GetConnection()
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, String srcTable)
   at JO_Monitoring_System.database1.LoadSQL(String mySql, String tblName) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Modules\Copy of database.vb:line 156
[06/17/2017 15:06:04] Authentication to host 'localhost' for user 'blade' using method 'mysql_native_password' failed with message: Access denied for user 'blade'@'localhost' (using password: YES)
[06/17/2017 15:06:04] Database: C@TCHM3
[06/17/2017 15:42:20] Host '192-164-0-92.adsl.highway.telekom.at' is not allowed to connect to this MariaDB server
[06/17/2017 15:42:20] Database: C@TCHM3
[06/17/2017 15:42:22] LoadSQL - MySql.Data.MySqlClient.MySqlException (0x80004005): Host '192-164-0-92.adsl.highway.telekom.at' is not allowed to connect to this MariaDB server
   at MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at MySql.Data.MySqlClient.NativeDriver.Open()
   at MySql.Data.MySqlClient.Driver.Open()
   at MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   at MySql.Data.MySqlClient.MySqlPool.GetConnection()
   at MySql.Data.MySqlClient.MySqlConnection.Open()
   at System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   at System.Data.Common.DbDataAdapter.Fill(DataSet dataSet, String srcTable)
   at JO_Monitoring_System.database1.LoadSQL(String mySql, String tblName) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Modules\Copy of database.vb:line 156
[06/17/2017 15:44:54] Unable to connect to any of the specified MySQL hosts.
[06/17/2017 15:44:54] Database: C@TCHM3
[06/23/2017 13:47:46] REPORT - Microsoft.Reporting.WinForms.LocalProcessingException: An error occurred during local report processing. ---> System.ApplicationException: The report definition for report 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc' has not been specified ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Microsoft.ReportingServices.StandalonePreviewStore.GetReportDefinition(ReportID reportId)
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   at Microsoft.Reporting.LocalService.GetCompiledReport(CatalogItemContextBase itemContext, Boolean rebuild, ControlSnapshot& snapshot)
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   at Microsoft.Reporting.WinForms.LocalReport.SetParameters(IEnumerable`1 parameters)
   at JO_Monitoring_System.frmReport.ReportInit(String mySql, String dsName, String rptUrl, Dictionary`2 addPara, Boolean hasUser) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Rports\frmReport.vb:line 83
[06/23/2017 13:48:10] REPORT - Microsoft.Reporting.WinForms.LocalProcessingException: An error occurred during local report processing. ---> System.ApplicationException: The report definition for report 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc' has not been specified ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Microsoft.ReportingServices.StandalonePreviewStore.GetReportDefinition(ReportID reportId)
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   at Microsoft.Reporting.LocalService.GetCompiledReport(CatalogItemContextBase itemContext, Boolean rebuild, ControlSnapshot& snapshot)
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   at Microsoft.Reporting.WinForms.LocalReport.SetParameters(IEnumerable`1 parameters)
   at JO_Monitoring_System.frmReport.ReportInit(String mySql, String dsName, String rptUrl, Dictionary`2 addPara, Boolean hasUser) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Rports\frmReport.vb:line 83
[06/23/2017 13:48:29] REPORT - Microsoft.Reporting.WinForms.LocalProcessingException: An error occurred during local report processing. ---> System.ApplicationException: The report definition for report 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc' has not been specified ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Microsoft.ReportingServices.StandalonePreviewStore.GetReportDefinition(ReportID reportId)
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   at Microsoft.Reporting.LocalService.GetCompiledReport(CatalogItemContextBase itemContext, Boolean rebuild, ControlSnapshot& snapshot)
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   at Microsoft.Reporting.WinForms.LocalReport.SetParameters(IEnumerable`1 parameters)
   at JO_Monitoring_System.frmReport.ReportInit(String mySql, String dsName, String rptUrl, Dictionary`2 addPara, Boolean hasUser) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Rports\frmReport.vb:line 83
[06/23/2017 14:01:07] REPORT - Microsoft.Reporting.WinForms.LocalProcessingException: An error occurred during local report processing. ---> System.ApplicationException: The report definition for report 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc' has not been specified ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\bin\Debug\Reports\rpt_JobOrderStatus.rdlc'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Microsoft.ReportingServices.StandalonePreviewStore.GetReportDefinition(ReportID reportId)
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.PreviewStore.GetCompiledReport(CatalogItemContextBase context, Boolean rebuild, Byte[]& reportDefinition, ControlSnapshot& snapshot)
   at Microsoft.Reporting.LocalService.GetCompiledReport(CatalogItemContextBase itemContext, Boolean rebuild, ControlSnapshot& snapshot)
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   --- End of inner exception stack trace ---
   at Microsoft.Reporting.WinForms.LocalReport.EnsureExecutionSession()
   at Microsoft.Reporting.WinForms.LocalReport.SetParameters(IEnumerable`1 parameters)
   at JO_Monitoring_System.frmReport.ReportInit(String mySql, String dsName, String rptUrl, Dictionary`2 addPara, Boolean hasUser) in C:\Users\MISJunmar\Documents\GitHub\JO Monitoring System\JO Monitoring System\Rports\frmReport.vb:line 83
[06/23/2017 14:28:35] Authentication to host 'MISGwapohon-PC' for user 'ellie' using method 'mysql_native_password' failed with message: Reading from the stream has failed.
[06/23/2017 14:28:35] Database: J0C@TCH
